mariamiah/The-Green-Mile
# #168692066: set up travisCI

#### What does this PR do?
Adds validation for the user details

#### Description of the task to be implemented.
- install nose
- add a travis.yml file

#### How should this be manually tested?
- Fetch this branch by running this command `git fetch origin ch-setup-travis-168692066`
- Check into this branch by running `git checkout ch-setup-travis-168692066`
- Navigate to the virtual environment `source venv/bin/activate`
- Install requirements by running `pip install -r requirements.txt`
- Run tests by running `nosetests <path-to-test-file>

#### Related Pivotal Tracker stories.
[#68692066](https://www.pivotaltracker.com/story/show/168692066)
